src/write.c

```diff
@@ -1642,7 +1634,9 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
         avifEncoderItem * item = &encoder->data->items.item[itemIndex];
         if (item->codec) {
             const avifImage * cellImage = cellImages[item->cellIndex];
+            avifImage * cellImagePlaceholder = NULL; // May be used as a temporary, modified cellImage. Left as NULL otherwise.
```

Comment:
It seems that the variable `paddedCellImage` is renamed `cellImagePlaceholder` and its declaration is moved forward to here, correct?

It seems that the original location of its declaration is better because it is right before the first use. Is the new location better when the Sample Transforms code is added?

Reply:
> Is the new location better when the Sample Transforms code is added?

I would say so. It is close to `cellImage` which may point to `cellImagePlaceholder`.